tests/test_plot.py

```diff
@@ -29,24 +29,34 @@ def test_show_raster():
             show((src, 1))
             fig = plt.gcf()
             plt.close(fig)
+
         except ImportError:
             pass
 
-        with rasterio.open('tests/data/RGB.byte.tif') as src:
-            try:
-                show(src)
-                fig = plt.gcf()
-                plt.close(fig)
-            except ImportError:
-                pass
+    with rasterio.open('tests/data/RGB.byte.tif') as src:
+        try:
+            show((src, (1, 2, 3)))
+            fig = plt.gcf()
+            plt.close(fig)
+
+        except ImportError:
+            pass
+
+    with rasterio.open('tests/data/RGB.byte.tif') as src:
+        try:
+            show(src)
+            fig = plt.gcf()
+            plt.close(fig)
+        except ImportError:
+            pass
```

Comment:
@choldgraf would you be willing to break this up into multiple tests? One `show()` per test?